index.html

```diff
@@ -16,20 +16,30 @@
 
 <body>
     <header>
-        <nav class="margin flex-row">
+        <nav class="margin flex-row nav-query">
             <h2>My logo</h2>
-            <img id="menu-button" src="images/Enabled.png" alt="Menu Button">
+            <ul class="desktop-menu flex-row">
+                <li>
+                    <p>Portfolio</p>
+                </li>
+                <li>
+                    <p>About</p>
+                </li>
+                <li>
+                    <p>Contact</p>
+                </li>
+            </ul>
+            <button id="menu-button"></button>
         </nav>
     </header>
     <section class="intro flex-column">
-        <h1 class="margin">I'm Gabriel. <br> Glad to see <br> you!</h1>
-        <p class="margin">I’m a software developer! I can help you build a product , feature or website Look through
-            some of my
-            work
-            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact
-            me.
+        <h1 class="margin h1-query intro-margin">I'm Gabriel. <br>Glad to see you!</h1>
+        <p class="margin intro-margin description">I’m a software developer! I can help you build a product , feature or
+            website
+            Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t hesitate to contact me.
```

Comment:
- [ ] Great job on your project! I noticed an issue **with the alignment when switching from desktop to mobile view**. The content is pushed **to the left and there's an empty space on the right side in desktop view**. I took a look at your code and it seems like the issue is caused by the `width` of one of your hidden images set to `93%` `(style.css - line 153)`. To fix this, you can remove the `width` or adjust it to a smaller value. Additionally, try to avoid using `vw` units for sizes and instead use more fixed units like **pixels or percentages**. This **will help maintain the consistency of your layout across different screen sizes**. Keep up the good work! 🎯

  _Watch the next video for a better understanding_ 🎬
  
  [scrnli_5_7_2023_12-47-55 AM.webm](https://user-images.githubusercontent.com/107952724/236660194-4f56b7c3-500e-43cb-a321-a87ef5cb1f69.webm)

Reply:
Hey! I really appreciate the review, I was not sure how to address it.
I also set the width and height of desktop images to 0 while on mobile ratios.

Reply:
Issue addressed!